### PR TITLE
Fix the inhirit flag

### PR DESCRIPTION
**Pull request by jaroel**
inherit flag equals to 'not \_\_ac_local_roles_block__'

### CODE REVIEW

**Comment by tisto** [comment]
@sneridagh could you pls have a look and merge if you are ok with it?

**Comment by tisto** [comment]
Actually I would like to wait with merging and do a "breaking changes" release after the next one.

File: CHANGES.rst [hunk 1]
```diff
@@ -23,6 +29,7 @@ Bugfixes:
 - Fix the @move endpoint fails to return 403 when the user don't have proper
   delete permissions over the parent folder.
   [sneridagh]
+>>>>>>> master
```
Comment (lukasgraf) [inline CHANGES.rst]: Mege fail

Comment (tisto) [inline CHANGES.rst]: @lukasgraf thx! fixed.

**Pull request merged**